app/experimenter/experiments/tasks.py

```diff
@@ -71,9 +80,14 @@ def create_experiment_bug_task(user_id, experiment_id):
         logger.info("Bugzilla ticket notification sent")
         raise e
 
+    metrics.incr("create_experiment_bug.completed")
```

Comment:
I think I'd rather have this above in the `try` block so it's only sent if the task completes successfully, otherwise to find the number of successful completes you need to do `completed - failed` right?

Reply:
That makes sense. It would then follow the other code as well. The way I read it the exception would `raise` so that completed `incr` wouldn't get sent anyway. But it follows more logically if it was in the try block. 👍